Formula/imgcat.rb

```diff
@@ -0,0 +1,18 @@
+class Imgcat < Formula
+  desc "Displays one or more images inline at their full size."
+  homepage "https://www.iterm2.com/documentation-images.html"
+  url "https://raw.githubusercontent.com/gnachman/iTerm2/master/tests/imgcat"
```

Comment:
So, I created a repo and versionned it, but then I get:

```
imgcat:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
  * GitHub repository too new (<30 days old)
Error: 2 problems in 1 formula
```

So should I just leave it for later to see if [Dorian/imgcat](https://github.com/Dorian/imgcat) takes off?